sagemaker-featurestore/feature_store_kms_key_encryption.ipynb

```diff
@@ -0,0 +1,593 @@
+    "            },\n",
+    "            \"Action\": \"kms:GenerateDataKey*\",\n",
+    "            \"Resource\": \"*\",\n",
+    "            \"Condition\": {\n",
+    "                \"StringLike\": {\n",
+    "                    \"kms:EncryptionContext:aws:cloudtrail:arn\": [\n",
+    "                        \"arn:aws:cloudtrail:*:123456789012:trail/*\",\n",
+    "                        \"arn:aws:cloudtrail:*:123456789012:trail/*\"\n",
+    "                    ]\n",
+    "                }\n",
+    "            }\n",
+    "        },\n",
+    "        {\n",
+    "            \"Sid\": \"Enable CloudTrail log decrypt permissions\",\n",
+    "            \"Effect\": \"Allow\",\n",
+    "            \"Principal\": {\n",
+    "                 \"AWS\": [role]\n",
+    "            },\n",
+    "            \"Action\": \"kms:Decrypt\",\n",
+    "            \"Resource\": [\"*\"],\n",
+    "            \"Condition\": {\n",
+    "                \"Null\": {\n",
+    "                    \"kms:EncryptionContext:aws:cloudtrail:arn\": \"false\"\n",
+    "                }\n",
+    "            }\n",
+    "        }\n",
+    "    ]\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Create your new KMS key using the policy above and your KMS client. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    new_kms_key = kms.create_key(\n",
+    "    Policy=json.dumps(policy),\n",
+    "    Description='string',\n",
+    "    KeyUsage='ENCRYPT_DECRYPT',\n",
+    "    CustomerMasterKeySpec='SYMMETRIC_DEFAULT',\n",
+    "    Origin='AWS_KMS'  \n",
+    "    )\n",
+    "    AliasName = 'my-new-kms-key' ## provide a unique alias name\n",
+    "    kms.create_alias(AliasName='alias/'+AliasName, TargetKeyId=new_kms_key['KeyMetadata']['KeyId'])\n",
+    "    print(new_kms_key)\n",
+    "except Exception as e:\n",
+    "    print(\"Error {}\".format(e))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that we have our KMS key created and the necessary operations added to our role, we now load in our data. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "customer_data = pd.read_csv(\"data/feature_store_introduction_customer.csv\")\n",
+    "orders_data = pd.read_csv(\"data/feature_store_introduction_orders.csv\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "customer_data.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "orders_data.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "customer_data.dtypes"
```

Comment:
Maybe we should also print out data types of orders

Reply:
Added `orders_data.dtypes`